### PR TITLE
L9: Model serving

### PR DESCRIPTION
**Pull request by ko3a4ok**
- [x] Web UI
- [x] REST API
- [x] Deployments

### CODE REVIEW

File: L9/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM huggingface/transformers-pytorch-cpu
```
Comment (truskovskiyk) [inline L9/Dockerfile]: add tag please

File: L9/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+streamlit
+torch
+sentence-transformers
+scipy
+fastapi
+uvicorn[standard]
+pytest
+httpx
```
Comment (truskovskiyk) [inline L9/requirements.txt]: add versions

File: L9/tests/context.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import os
+import sys
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.dirname(SCRIPT_DIR))
```
Comment (truskovskiyk) [inline L9/tests/context.py]: just use PYTHONPATH for this

File: L9/web/web.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+def get_model():
+  return SentenceTransformer('TaylorAI/bge-micro')
+
+
+data_load_state = st.text('Loading data...')
+
+
+@st.cache(allow_output_mutation=True)
+def load_data():
+  df = pd.read_fwf(DATA_URL).iloc[:1000, :1]
```
Comment (truskovskiyk) [inline L9/web/web.py:21]: remove hardcode

**Review by truskovskiyk — APPROVED** [review]

File: L9/tests/context.py (hunk 3 above)
Comment (ko3a4ok) [reply L9/tests/context.py]: Great, thanks! it works now with just:
```shell
 PYTHONPATH=`pwd` pytest
```

File: L9/web/web.py (hunk 4 above)
Comment (ko3a4ok) [reply L9/web/web.py:21]: the data file is too big. It would take over 15 minutes to generate embedding for the whole content

**Pull request merged**